### ISSUE TITLE
smite implement

### ISSUE THREAD

**Issue by marcocorriero** [TotalFreedom/TotalFreedomMod#1776]
smite need to say who smites the player.

Ex, prozza already coded Smite reasons, but the smite will stay anonymous,

IF a rcon breaches?
If a telnet go to rogue?

Smite need to say in reason

The admin smites it.

I can help.

**Comment by marcocorriero** [comment]
Idk how to wire it, but i did the code,

https://github.com/TotalFreedom/TotalFreedomMod/compare/tfm50...marcocorriero:patch-1?diff=split&name=patch-1

**Comment by ShadowArmadyl** [comment]
If you need to know who smited the player, that's what telnet is for.

This is unneeded.

**Comment by marcocorriero** [comment]
@ShadowArmadyl is needed, if someone do sign exploit-smite? It will make rogue admins traceable, telnet commands is untraceable...

**Comment by RobinGall2910** [comment]
@marcocorriero that wouldn't work... if you're trying to say that sign exploits would find out who's done them, that wouldn't work. It would only send who tried, as in the idiot.

**Comment by Wild1145** [comment]
I think that this should be added, for no other reason than it is useful to know quickly who smited who, I cannot be bothered getting on telnet, and as an OP they dont have access to telnet, but should know who smited them.

**Comment by trustthepie** [comment]
Agreed with @Wild1145. Normally ops are like "Who smited me?" Perhaps we need to make smite reasons and executor name more noticeable because no one sees them?
